apps/asap-server/test/controllers/research-outputs.test.ts

```diff
@@ -609,6 +614,50 @@ describe('ResearchOutputs controller', () => {
       });
     });
   });
+
+  describe('create', () => {
+    afterEach(() => {
+      expect(nock.isDone()).toBe(true);
+    });
+
+    afterEach(() => {
+      nock.cleanAll();
+    });
+    test('creating the research output should return the id from squidex rest', async () => {
```

Comment:
I'm sure that's what squidex REST does but I thought maybe you considered returning the entire new object instead?

Reply:
btw there is a lot more going on here than just testing the return - i think it's good to create another test where you validate the input that is send to the client, for instance i can see that although our input only contains the type the payload has two extra fields (which im sure is intended)

Reply:
> I'm sure that's what squidex REST does but I thought maybe you considered returning the entire new object instead?

We wanted to return the whole object, but we can't. The Research Output is on draft.
 
Any suggestions welcome :)
But we thought this could wait, until we start editing the RO

Reply:
you can actually fetch the drafts from squidex too. The only thing i would say is that you could do it now so you dont have to change the interface later. Maybe return an object with an ID, rather than just ID? so that later you can add the rest of the fields?